projects/ad7768-1fmcz/src/ad77681evb.c

```diff
@@ -135,6 +143,17 @@ int main()
 	Xil_ICacheEnable();
 	Xil_DCacheEnable();
 
+	ret = axi_clkgen_init(&clkgen, &clkgen_init);
+	if (ret != SUCCESS) {
+		printf("error: axi_clkgen_init() failed\n");
+		return FAILURE;
+	}
+
+	ret = axi_clkgen_set_rate(clkgen, 80000000);
```

Comment:
If AD77681_SPI_ENG_REF_CLK_FREQ_HZ will be used, shouldn't be used here too?

Reply:
See above comment:

https://github.com/analogdevicesinc/no-OS/pull/795#discussion_r507604399